pkg/nfd-master/nfd-master.go

```diff
@@ -126,6 +133,15 @@ func NewNfdMaster(args *Args) (NfdMaster, error) {
 			return nfd, fmt.Errorf("-ca-file needs to be specified alongside -cert-file and -key-file")
 		}
 	}
+	if args.DenyLabelNs == nil {
+		args.DenyLabelNs = make(utils.StringSetVal)
+	}
+	// We should forcibly deny *.kubernetes.io
```

Comment:
Remove `Should` bc we do 
```suggestion
	// We forcibly deny *.kubernetes.io
```